### PR TITLE
add: docs - provide instructions on how to contribute

### PR DESCRIPTION
**Pull request by benkoshy**
### What is this?
Provides instructions on how to contribute, especially if adding files are required.

### Why?
Because this repository is a little tricky - there are 9 separate projects and new comers may get confused. (Also it is a reference for myself ;D).

### CODE REVIEW

**Comment by wtertinek** [comment]
Thanks for this contribution, it's good to point that out.

File: README.md [hunk 1]
```diff
@@ -86,6 +86,19 @@ Please contribute to this project by
...
+
+The best way to do this is to Open the (.csproj) in a text editor and to add the following lines (amending the file names and paths):
+
+```xml
+<Compile Include="..\..\Sources\Linq2Acad\Extensions\GroupExtensions.cs">
+  <Link>Extensions\GroupExtensions.cs</Link>
+</Compile>
+```
+
+![alt How to edit a .csproj file](./docs/addSources.PNG)
```
Comment (wtertinek) [inline README.md:100]: In my opinion the screenshot is not really necessary. Editing a file shouldn't need further description.

**Comment by benkoshy** [comment]
@wtertinek  Force pushed and rebased on top of patch-4 - so you should not have trouble merging.

**Comment by wtertinek** [comment]
Thanks Ben, looks good now. One remark: maybe it is a good idea to move the contents of the `Contributing` section to a CONTRIBUTING.md file and only like to that file from README.md? Keeps README.md clear. What do you think?

**Pull request merged**